server-master/src/main/scala/com/aliyun/emr/rss/service/deploy/master/Master.scala

```diff
@@ -79,16 +79,21 @@ private[deploy] class Master(
     // blacklist worker count
     source.addGauge(MasterSource.BlacklistedWorkerCount,
       _ => statusSystem.blacklist.size())
+    val clusterSlotsUsageLimit: Double = RssConf.clusterSlotsUsageLimitPercent(conf)
     // worker count
     source.addGauge(MasterSource.WorkerCount,
       _ => statusSystem.workers.size())
-    val (totalSlots, usedSlots, overloadWorkerCount) = getClusterLoad
     // worker slots count
-    source.addGauge(MasterSource.WorkerSlotsCount, _ => totalSlots)
+    source.addGauge(MasterSource.WorkerSlotsCount,
+      _ => statusSystem.workers.asScala.map(_.numSlots).sum)
```

Comment:
Variable "statusSystem.workers" includes workers on blackslist. I think using "workersSnapShot" will be better.

Reply:
> Variable "statusSystem.workers" includes workers on blackslist. I think using "workersSnapShot" will be better.

Done